packages/angular/golden/clr-angular.d.ts

```diff
@@ -537,8 +538,8 @@ export declare class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<
     customFilter: boolean;
     get field(): string;
     set field(field: string);
-    get filterValue(): string | [number, number];
-    set filterValue(newValue: string | [number, number]);
+    get filterValue(): string | [number, number] | any;
```

Comment:
if this can be `any`, why don't we list it as `[number, number] | any`? why is `string` important to call out here?

also, can it _really_ be any?

Reply:
I agree with Scott. Also, I believe we are adding more issues than we are fixing with setting up `any` as a possible type.